pages/blog/2022-summary.md

```diff
@@ -0,0 +1,252 @@
+
+### Google Search Console
+
+I believe that numbers from GSC can show how many more people hear about AsyncAPI and gain interest in the topic. In other words, the assumption is that the more people google, the more people are aware of AsyncAPI. And yes, the below numbers of impressions indicate a number that is higher than a milion :wink:
+
+<Figure
+  src="/img/posts/2022-summary/google-search.webp"
+/>
+
+In 2022 we didn't perform any specific modifications on the website to improve SEO. So the only explanation for the increase in impressions and clicks is the increased volume of documentation I wrote about in the previous section. More useful content, better ranking in search.
+
+## GitHub
+
+Like on other channels, we notice growing traffic in [our GitHub organization](https://github.com/asyncapi).
+
+### Total contributions
+
+In the last three years, we increased from 301 contributors in Q1 2020 to 1520 in Q4 2022.
+
+From 301 to 1520 contributors for things like commits, pull requests authoring and review, or issues participation. In just three years!!!
+
+<Figure
+  src="/img/posts/2022-summary/github-contrib.webp"
+/>
+
+This huge difference also has a negative aspect. More contributions mean more maintenance work. 
+
+### Negative aspect of growth
+
+Unfortunately, the number of maintainers does not grow as much as the number of contributors. This means that the difference between the number of submitted and resolved issues will get larger and larger.
+
+<Figure
+  src="/img/posts/2022-summary/github-issues.webp"
+/>
+
+Three years ago, we dreamed about having thousands of contributions. We forgot to balance that number with the proper number of maintainers. Now we need to react and make sure we also get a large number of active maintainers.
+
+## npm
+
+Most of our tools are written in JavaScript/TypeScript; we host them on Node Package Manager (npm).
+
+### Top packages
+
+Our most essential packages are [the project that provides AsyncAPI JSON Schema files](https://www.npmjs.com/package/@asyncapi/specs), [the Generator](https://www.npmjs.com/package/@asyncapi/generator), [the React component](https://github.com/asyncapi/asyncapi-react) and [the Parser](https://www.npmjs.com/package/@asyncapi/parser). Below you can see how downloads of these packages increased.
+
+<Figure
+  src="/img/posts/2022-summary/npm-stats.webp"
+/>
+
+You do not believe in this data? Lemme repeat what I wrote last year:
+
+> The increase in numbers is insanely huge. People automatically say that it is because of build systems, local development, or some single corporation started using it at scale. In other words, the increase could result from just one project that uses these tools as a dependency.
```

Comment:
```suggestion
> The increase in numbers is insanely huge. People automatically say that it is because of build systems, local development, or that some single corporation started using it at scale. In other words, the increase could result from just one project that uses these tools as a dependency.
```